aeron-driver/src/main/c/aeron_driver_conductor.c

```diff
@@ -4635,11 +4636,17 @@ void aeron_driver_conductor_on_create_publication_image(void *clientd, void *ite
             image->log_file_name_length,
             image->log_file_name) < 0)
         {
-            return;
+            goto error_cleanup;
         }
     }
 
     aeron_driver_receiver_proxy_on_add_publication_image(conductor->context->receiver_proxy, endpoint, image);
+    return;
+
+error_cleanup:
+    AERON_APPEND_ERR("%s", "Failed aeron_driver_conductor_on_create_publication_image for %d %d", command->stream_id, command->session_id);
+    aeron_driver_conductor_log_error(conductor);
+    aeron_driver_receiver_proxy_on_remove_init_in_progress(conductor->context->receiver_proxy, endpoint, command->session_id, command->stream_id);
```

Comment:
Potentially I should delay the actual removal, to stop excess failed attempts if the error condition is persistent

Reply:
This is probably not too much of an issue if the error trace is the same for each error, i.e. the same logged parameters and call error stack, then the logger will collapse it into a single log entry and maintain a count.